dbms/src/DataStreams/NonJoinedBlockInputStream.cpp

```diff
@@ -232,9 +227,19 @@ size_t NonJoinedBlockInputStream::fillColumns(const Map & map,
         setNextCurrentNotMappedRow();
         if (rows_added == max_block_size)
         {
+            /// Fill left columns with defaults
+            for (size_t j = 0; j < num_columns_left; ++j)
+                mutable_columns_left[j]->insertManyDefaults(rows_added);
```

Comment:
use `break` here then we can merge L230-L232 and L236-L242?

Reply:
Update it.